app/reducers/settings.js

```diff
@@ -119,3 +125,17 @@ function isoValidationComplete(state, action) {
     isoValidationState: action.payload.isValid,
   };
 }
+
+function toggleConfirmDialog(state, action) {
```

Comment:
These seem more like sets than toggles. Toggle to me kind of implies the current state is switched.